src/control/server/instance_storage.go

```diff
@@ -99,6 +99,23 @@ func (ei *EngineInstance) awaitStorageReady(ctx context.Context, skipMissingSupe
 	}
 	ei.log.Debugf("needsScmFormat: %t", needsScmFormat)
 
+	if !needsMetaFormat && ei.storage.ControlMetadataPathConfigured() {
+		cfg, err := ei.storage.GetScmConfig()
+		if err != nil {
+			return err
+		}
+		if (cfg.Class == "ram") && ei.storage.BdevRoleMetaConfigured() {
+			ei.log.Debugf("scm class is ram and bdev role meta configured")
+			err := ei.storage.FormatScm(true)
+			if err != nil {
+				ei.log.Errorf("instance %d: failed to setup the scm: %s", idx, err)
```

Comment:
Why wouldn't this be fatal? Any reason to believe trying it again would work?

Reply:
Instead of handling the failure here, I allow the code to fall through and take the default code path where it requires user to explicitly invoke format command.

Reply:
But we've already attempted the format here, and it failed. I guess I'm wondering what value there is in trying again, instead of having the server just error and quit.